### PR TITLE
Update vanilla dep to v2.0.1

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated the docs to v2.0.1

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check the docs pages in-browser at different viewports

### CODE REVIEW

**Comment by kwm14** [comment]
## Design review
Looking good @deadlight just a couple of minor updates 😉 
- [ ] Move `hr` rule under page title on **/building-vanilla** and **/customising-vanilla** so it's consistent with the rest of pages across the site
- [ ] Add `br` under the table to give space between heading and title on **/settings/font-settings**

Screenshot 2019-06-11 at 16 29 28

**Comment by deadlight** [comment]
@kwm14 all fixed

**Comment by kwm14** [comment]
Thanks, @deadlight LGTM 👍

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ before_script:
 script:
   - yarn run test
   - yarn run build-docs
-  - percy snapshot --widths "375,1280" docs/_site/
+  - percy snapshot --widths "375,1280,1600" docs/_site/
```
[the remark this answers is not in the document]
Comment (deadlight) [reply .travis.yml]: It'll be handy if we do something with unintended consequences. I'll keep this config unless it causes any problems.

**Comment by deadlight** [comment]
@anthonydillon fixed up the branch. I left some notes in your original comment above (which I have turned into a checklist)

File: .travis.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anthonydillon) [reply .travis.yml]: Sorry, I dont understand. We used to Percy examples not the docs. Now we only check differences in docs which does not use the local Vanilla codebase and does not include the template examples used to complex tests.

[the remark this answers is not in the document]
Comment (deadlight) [reply .travis.yml]: Oh right, I forgot that it doesn't render the full examples.

We can update that then

**Comment by deadlight** [comment]
@anthonydillon I can't see the checklist now... GitHub has removed it, which is pretty annoying.

I didn't add gulp back, that file was already there and was caught in the find/replace of /en/... I guess I can just remove that

**Comment by deadlight** [comment]
@anthonydillon Your original comments (recovered from the email):


    The typo table is showing mark up
    The related links at the bottom of the docs pages 404
    Do we need Legal info in the footer?
    On page utilities/floats/ the floats look full width.
    /patterns/navigation/ has back ticked content which I assume should be code
    The Vertical divider example is not loading on lists docs
    There is a stray </a> at the bottom of the /patterns/lists/ page
    The Social icons in the docs do not work
    We should add Lyubo to the contribute page

I can't see any problems with the social icons, they show up and change on hover.

I've created a PR on the website repo to add Lyubo (and anyone else who makes sense) to the /contributors page

**Comment by deadlight** [comment]
@anthonydillon also yes we need the legal notification - we gather user data, use cookies, etc.. All that's covered in the legal pages.

**Pull request merged**